.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "azure-mock/data"]
+	path = azure-mock/data
+	url = git@github.com:Hochfrequenz/machine-readable_anwendungshandbuecher.git
```

Comment:
macht das nicht mit gitkraken probleme?
```suggestion
	url = https://github.com/Hochfrequenz/machine-readable_anwendungshandbuecher
```

Reply:
nope worked for me ^^

Reply:
Ich habe bei mir aber auch einen ssh-agent konfiguriert.
Kannst du mal schauen ob das bei klappt?